Modules/_operator.c

```diff
@@ -942,6 +950,7 @@ static struct PyMethodDef operator_methods[] = {
     _OPERATOR_GE_METHODDEF
     _OPERATOR__COMPARE_DIGEST_METHODDEF
     _OPERATOR_LENGTH_HINT_METHODDEF
+    {"_idfunc",         (PyCFunction)_operator__idfunc, METH_O},
```

Comment:
I don't know adding `operator. _idfunc` is a great idea, but if it needs to be added.
I would like to suggest using the [clinic tools](https://docs.python.org/3/howto/clinic.html).

On the other hand, instead of adding an `operator. _idfunc` how about implementing `typing.NewType` itself
as the C accelerated implementation. (datetime module has a similar structure for those cases)

Reply:
Making all of NewType implemented in C would make the implementation harder to work with and be more complicated to review. NewType has only one component that actually has a significant runtime impact with normal usage, and it's the `__call__` method, which as it happens we can implement as the simplest possible function. So I'd recommend against implementing all of it in C.

Reply:
@JelleZijlstra 

Apparently, adding the `operator. _idfunc` is the easiest way to enhance NewType.__call__ method.
But is there any case that adding those kinds of functions to the accelerated specific modules?

How about adding the Modules/_typing.c for those accelerating?
For this case, the statistics module is a good use case.
https://github.com/python/cpython/blob/main/Modules/_statisticsmodule.c

if we add the operator._idfunc into pure python module it will expose to other alternative python interpreter implementation which even does not need such accelerating.

Reply:
So my conclusion is adding accelerating utility function +1
But the location is -1

Reply:
Yes, I think it's hard to come up with a good place for this. Adding a new C module for just this function feels like overkill, but maybe that's what we have to do.

Reply:
> Adding a new C module for just this function feels like overkill,

FYI `_statistics` module only exists for `_statistics._normal_dist_inv_cdf` so this case also looks okay :)